tests/end_to_end/test_vulnerability_detector/test_vulnerability_detector.py

```diff
@@ -139,6 +133,12 @@ def setup_vulnerability_tests(host_manager: HostManager) -> Generator:
     logger.error("Restarting managers")
     host_manager.control_environment('restart', ['manager'])
 
+    logger.error("Save the Wazuh indexer username and password into the Wazuh manager keystore")
```

Comment:
Shouldn't it be `info` or `debug` instead of `error`?

Reply:
We have not determinate the logic of the logging. I suggest the following:
- **Critical**: Logging messages from the tests
- **Error**: Logging messages from the fixtures
- **Info**: Logging messages from the rest of the libraries
- **Debug**: Very verbose logging messages like testinfra/ansible debug messages

Reply:
We will revisit this in the future